nixos/modules/services/databases/hbase.nix

```diff
@@ -35,16 +35,12 @@ in {
   ###### interface
 
   options = {
+    services.hbase-standalone = {
```

Comment:
the module file should be renamed as well, and we'll need a few `mkRenamedOptionModule` for these.